app/components/search/SearchResults/SearchResults.tsx

```diff
@@ -215,6 +218,9 @@ const SearchResult = ({
   }
 
   const toggleTextingModal = () => setTextingIsOpen(!textingIsOpen);
+  // TODO: this bookmarkAdded boolean should be set in accordance with the value of the bookmark model
+  // returned by the API or just read off of the `hit` directly; it is set here for dev purposes
+  const bookmarkAdded = false;
```

Comment:
Oh right, yes! Thank you. I updated my comments to reflect that this will need to be fetched from the API. That would be quite a lot of calls to the API, 20 per page. Do you think it'd be more performant if the API had an endpoint that returned all of a users' bookmarks per service/resource ID? Each `hit` could check against this list to determine whether it is bookmarked or not